Set of fixes for away mission maps.

CountAlex opened the pull request:
Fixing amount little things for away missions:
- properly coded navigation lights at magshield;
- generic plushie at magshield changed to random plushie;
- casino ship now has few little light sources for better exposition;
- casino ship has a source of some !!FUN!! in a place you should expect;
- smugglers base also has a reasonable amount of !!FUN!! now to be less of a lootfest;
- marooned officer crashsite now will have slightly colder atmosphere, day light in open areas and hopefully actually present atmosphere.

PsiOmegaDelta commented:
None of the maps look like they were cleaned.

CountAlex commented:
Which is weird since I definetely did that before pushing. I'll do that again to be sure.

CountAlex commented:
Maps definetely were cleaned with jmerge and my mind doesn't play tricks with me since additional attempts yield no result to push.

PsiOmegaDelta commented:
If you're running the cleanup tool after a commit it won't have much of an effect, unless the old backup files from step 1 still remain.

CountAlex commented:
(EDIT: I know) I've runned the tool before commiting because I had to stash few maps (bearcat, delerict etc.) that were not cleaned.

PsiOmegaDelta committed:
Set of fixes for away mission maps.

Pcoded navigation lights at magshield;
Generic plushie at magshield changed to random plushie;
Casino ship now has few little light sources for better exposition;
Casino ship has a source of some !!FUN!! in a place you should expect;
Smugglers base also has a reasonable amount of !!FUN!! now to be less of a lootfest;
Marooned officer crashsite now will have slightly colder atmosphere, day light in open areas and hopefully actually present atmosphere.

PsiOmegaDelta commented:
Map files cleaned.
